src/aiodynamo/client.py

```diff
@@ -73,6 +73,9 @@ class Table:
     name: TableName
 
     async def exists(self) -> bool:
+        """
+        Checks that the table exist.
```

Comment:
```suggestion
        Checks if the table exists.
```

Reply:
I think that Python docs use the imperative mood, for example, `os.path.exists(...)`:
>     exists(path)
>        Test whether a path exists.  Returns False for broken symbolic links